### ISSUE TITLE
a power tripping admin is blocking people for no reason on discord

### ISSUE THREAD

**Issue by NoUserNameForYou** [moonlight-stream/moonlight-qt#950]
Egotistical people like him sully the name of your software. FYI.

**Comment by goodonematey** [comment]
Yep, that would be `yungleballz`, so, am not gonna disclose anything about him, but I'd say it's an age thing. However, what I can tell you - based on any quick observation from anyone with more than 2 brain cells is what he is: a nobody. The "code & conduct moderation team" or people who change the CoC and then say they "code". I went through this guy's replies and he's always spitting out idiotic, dumb shit that seems connected to what's being discussed. He must've read some articles or talked to the devs here and there and, bang, he brings up terms that have nothing to do with the discussion at hand. **He creates so, so, so much noise, but sometimes, he's useful.** I guess even a broken clock is right a few times a day. Anyways, dude, let the adults talk and sit in your corner.

I was asking the Moonlight developers about the video pipeline of Windows itself and how exactly Windows itself provides the output data and how Moonlight captures it. I was thinking about it and, I realized - hey, it might not be Moonlight that's problematic, given the specifics of the issue and thinking through it. So, I thought,  why not go and ask the Sunshine project people? They'd probably know better, since they provide that output. Realistically, Moonlight only listens to some output and provides it for you, so, I went ahead and asked my question.

`yungleballz`, in his infinite wisdom, replies with "dude" in my Sunshine server thread, then deletes it. I'm flabergasted - I was just simply asking the Sunshine devs about this, because I realized where the problem is, most likely. Anyways, I ping him in the `off-topic` channel of Moonlight to tell him the above and I get a timeout. I wasn't attacking him personally, just his work. I get a timeout and then a ban...**on both servers.**

Now, I could be an asshole and also pay it back, just the way you did it (difference is, you have a fork and I have a nuclear weapon, so, perhaps treats others on the internet better, because it took me like 30s to pull all your data? So, you're not "anonymous", that connection you think gives that actually gets you closer to me), but the reality is, you're an absolute worm, a parasite, a nobody that wishes he knew how to code, but because you can't, you resort to simply being "customer support". Frankly, that's revenge enough.

In fact, if you keep being a "mod" (Jesus, imagine finding any goodness in this) is precisely the best payback, because I know that, every time you go to sleep, you remember how big a failure you are.

@cgutman Very nice guy you've got in here, my man. Absolutely not gonna explode one day.

**Comment by NoUserNameForYou** [comment]
> fork and I have a nuclear weapon

Hahaha. You weren't lying. You went nuclear on the power tripping, egotistical, unethical buffoon.
